### PR TITLE
[zh] Remove reviewer for kubeadm

### PR DESCRIPTION
**Pull request by Sea-n**
- Re-sync en content
- Remove reviewers from metadata
- Remove redundant spaces
- Add link anchors
- Change `master` to `control-plane node`

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join.md [hunk 1]
```diff
@@ -22,12 +27,12 @@ This command initializes a Kubernetes worker node and joins it to the cluster.
...
-`kubeadm join` bootstraps a Kubernetes worker node and joins it to the cluster.
-This action consists of the following steps:
+<!--
+`kubeadm join` bootstraps a Kubernetes worker node or a control-plane node and adds it to the cluster.
+This action consists of the following steps for worker nodes:
 -->
-`kubeadm join` 初始化 Kubernetes 工作节点并将其加入集群。
-该操作过程包含下面几个步骤：
+`kubeadm join` 初始化 Kubernetes 工作节点或控制平面节点并将其添加到集群中。
+该操作包括工作节点的以下步骤：
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join.md:35]:
```suggestion
对于工作节点，该操作包括以下步骤：
```

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join.md [hunk 2]
```diff
@@ -481,23 +490,25 @@ users: []
 
 * 关闭 `cluster-info` ConfigMap 的公开访问：
 
-  ```shell
-  kubectl -n kube-public delete rolebinding kubeadm:bootstrap-signer-clusterinfo
-  ```
+```shell
+kubectl -n kube-public delete rolebinding kubeadm:bootstrap-signer-clusterinfo
+```
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-join.md:495]: 以上保持两个空格的缩进

**Comment by ydFu** [comment]
LGTM
/lgtm

**Pull request merged**